cmd/kubeadm/app/util/system/kernel_validator.go

```diff
@@ -218,14 +220,14 @@ func (k *KernelValidator) getKernelConfigReader() (io.Reader, error) {
 		// config module and check again.
 		output, err := exec.Command(modprobeCmd, configsModule).CombinedOutput()
 		if err != nil {
-			return nil, fmt.Errorf("unable to load kernel module %q: output - %q, err - %v",
-				configsModule, output, err)
+			return nil, pkgerrors.Wrapf(err, "unable to load kernel module %q: output - %q, err - ",
```

Comment:
lets change the end to `output: %q, err`

Reply:
i want to change it to `unable to load kernel module: %q, output: %q, err`